Explorer/Assets/Scripts/Global/Tests/EditMode/RealmLaunchSettingsShould.cs

```diff
@@ -44,22 +44,6 @@ public void DoNotSetDevelopmentModeIfMissingLocalSceneParam()
             Assert.AreEqual(70, realmLaunchSettings.TargetScene.y);
         }
 
-        [Test]
-        public void ApplyStartingPositionFromAppArgs()
```

Comment:
Maybe we should move this test elsewhere instead of deleting it completely?

Reply:
That test is checking the initial position from the RealmLaunchSettings. It have been removed entirely from that domain

Reply:
I know, but do we still have that launch param tested elsewhere?

Reply:
Brought back